patches/minecraft/net/minecraft/item/Food.java.patch

```diff
@@ -0,0 +1,49 @@
+--- a/net/minecraft/item/Food.java
++++ b/net/minecraft/item/Food.java
+@@ -11,9 +11,9 @@
+    private final boolean field_221472_c;
+    private final boolean field_221473_d;
+    private final boolean field_221474_e;
+-   private final List<Pair<EffectInstance, Float>> field_221475_f;
++   private final List<Pair<java.util.function.Supplier<EffectInstance>, Float>> field_221475_f;
+ 
+-   private Food(int p_i50106_1_, float p_i50106_2_, boolean p_i50106_3_, boolean p_i50106_4_, boolean p_i50106_5_, List<Pair<EffectInstance, Float>> p_i50106_6_) {
++   private Food(int p_i50106_1_, float p_i50106_2_, boolean p_i50106_3_, boolean p_i50106_4_, boolean p_i50106_5_, List<Pair<java.util.function.Supplier<EffectInstance>, Float>> p_i50106_6_) {
+       this.field_221470_a = p_i50106_1_;
+       this.field_221471_b = p_i50106_2_;
+       this.field_221472_c = p_i50106_3_;
+@@ -42,7 +42,7 @@
+       return this.field_221474_e;
+    }
+ 
+-   public List<Pair<EffectInstance, Float>> func_221464_f() {
++   public List<Pair<java.util.function.Supplier<EffectInstance>, Float>> func_221464_f() {
```

Comment:
This breaks binary compatibility as it's a public function, any mod could be calling it. Bad idea if you you want to pull this into 1.14, or just for general mods in 1.14

Why not map the list to resolve the suppliers so you keep the old return type?